examples/minimal_v2.json

```diff
@@ -35,7 +35,8 @@
       "map"
     ],
     "display_defaults": {
-      "color_by": "country"
+      "color_by": "country",
+      "language": "ja"
```

Comment:
Can you change this back to leave out the language setting as it was before this PR, once you are done testing (if you make other changes) and ready to merge? I think we want to keep the example dataset as is. Thanks!